src/main/java/com/o19s/es/ltr/ranker/LtrRanker.java

```diff
@@ -60,5 +60,10 @@
          * Get the feature score
          */
         float getFeatureScore(int featureId);
+
+        /**
+        * Get the size of this FeatureVector
+        */
+        int size();
```

Comment:
perhaps not needed since you have the featureSet available

Reply:
good point, sorry for the oversight. fixed.